.circleci/do-rtl-build.sh

```diff
@@ -32,7 +32,19 @@ run "cp -r ~/.sbt  $REMOTE_WORK_DIR"
 
 TOOLS_DIR=$REMOTE_RISCV_DIR
 LD_LIB_DIR=$REMOTE_RISCV_DIR/lib
-if [ $1 = "hwacha" ] || [ $1 = "gemmini" ]; then
+
```

Comment:
Don't you want to build the tests after the config is built? Or am I missing something here?

Reply:
Why is that necessary?
If the tests fail to build, I would rather know sooner, rather than have to wait for the simulator to build.